spec/lib/generators/good_job/install_generator_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+require 'generators/good_job/install_generator'
+
+describe GoodJob::InstallGenerator, type: :generator do
+  after { remove_example_app }
+
+  it 'creates a migration for good_jobs table' do
+    create_example_app
+
+    install_good_job
```

Comment:
I think this line should be expanded inside of the Example. E.g. if you follow my other suggestions:

```
example_app_cmd "rails g good_job:install"
```

Reply:
How about naming the method `run_in_example_app` or something better?

Reply:
👍